docs/tutorials/advanced/strategy_transformers.rst

```diff
@@ -104,13 +104,13 @@ The library includes the following transformers:
     >>> ApologizingDefector = ApologyTransformer([D], [C])(axelrod.Defector)
     >>> player = ApologizingDefector()
 
-You can pass any two sequences in. In this example the player would apologize
-after two consequtive rounds of `(D, C)`::
+   You can pass any two sequences in. In this example the player would apologize
+   after two consequtive rounds of `(D, C)`::
 
-    >>> import axelrod
-    >>> from axelrod.strategy_transformers import ApologyTransformer
-    >>> ApologizingDefector = ApologyTransformer([D, D], [C, C])(axelrod.Defector)
-    >>> player = ApologizingDefector()
+       >>> import axelrod
+       >>> from axelrod.strategy_transformers import ApologyTransformer
+       >>> ApologizingDefector = ApologyTransformer([D, D], [C, C])(axelrod.Defector)
+       >>> player = ApologizingDefector()
 
 * :code:`DeadlockBreakingTransformer`: Attempts to break :code:`(D, C) -> (C, D)` deadlocks by cooperating::
 
```

Comment:
I added this to this PR just to fix the indentation for the docs. It's unrelated.